src/routes/MapExplorer/MapExplorer.scss

```diff
@@ -0,0 +1,179 @@
+.MapExplorer-container {
```

Comment:
These class names are going to be already CSS Modules hash and name scoped to `MapExplorer` so you could scope names with that assumption.

Reply:
yeah not sure how that happened